src/modules/sub-items/components/table-view/table.view.item.component.js

```diff
@@ -31,7 +31,7 @@ export default class TableViewItemComponent extends Component {
      * @memberof TableViewItemComponent
      */
     enablePriorityInput() {
-        this.setState((state) => ({ ...state, priorityInputEnabled: true }));
+        this.setState(() => ({ priorityInputEnabled: true }));
```

Comment:
We do not need callback here and in a few other places in this file and in other modified files.

Reply:
See comment above: https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/92#discussion_r202540313